server/src/test/java/com/broadcom/lsp/cobol/usecases/TestXmlQueryNotTerminatesSemanticAnalysis.java

```diff
@@ -48,7 +48,7 @@
           + "       ENVIRONMENT DIVISION.\n"
           + "       DATA DIVISION.\n"
           + "       LINKAGE SECTION.\n"
-          + "       01 {$*VARNAME} PIC 9.\n"
+          + "       01 VARNAME PIC 9.\n"
```

Comment:
Why are you exclude this variable definition?

Reply:
It is a leftover of the UseCaseEngine version of this test. At the moment, the preprocessing is not applied to this test.